provisioners/ansible/playbooks/apps/create-aws-resources.yaml

```diff
@@ -6,6 +6,20 @@
   connection: local
 
   tasks:
+    - name: Create AWS Region file
+      template:
+        src: ../../../../templates/ansible/cloudformation.j2
+        dest: ../../../../stage/awsregion.yaml
+      tags:
+        - create
+
+    - name: AWS ELB Account ID
+      include_vars:
+        file: ../../../../stage/awsregion.yaml
+        name: AWSELBAccountID
```

Comment:
Why is it necessary to use cloudformation.j2 and awsregion.yaml only to create a `AWSELBAccountID` parameter?

Reply:
same as above